runtime/jcl/uma/se9_exports.xml

```diff
@@ -65,6 +65,5 @@
 	<export name="Java_java_lang_StackWalker_walkWrapperImpl" />
 	<export name="Java_java_lang_StackWalker_getImpl" />
 	<export name="Java_java_lang_invoke_MethodHandles_findNativeAddress">
-		<include-if condition="spec.flags.opt_panama and spec.flags.opt_methodHandle" />
 	</export>
```

Comment:
Removing the `<include-if>` element will have the effect of enabling this `<export>` element (where it was not before). Please just remove the `<export>` element (lines 67-68).